src/index.ts

```diff
@@ -97,8 +103,8 @@ export {default as PointerBox} from './PointerBox'
 export type {PointerBoxProps} from './PointerBox'
 export {default as Popover} from './Popover'
 export type {PopoverProps, PopoverContentProps} from './Popover'
-// export {default as Portal, registerPortalRoot} from './Portal'
-// export type {PortalProps} from './Portal'
+export {default as Portal, registerPortalRoot} from './Portal'
+export type {PortalProps} from './Portal'
```

Comment:
Looks safe to uncomment :)

https://github.com/primer/react/commit/bf114b042383966b98b90224fa9cac5e841d3173